pkgs/by-name/sd/sdrpp/package.nix

```
@@ -15,82 +15,88 @@
   airspy,
   airspyhf_source ? true,
   airspyhf,
-  bladerf_source ? true,
+  audio_source ? true,
+  bladerf_source ? stdenv.hostPlatform.isLinux,
   libbladeRF,
   file_source ? true,
   hackrf_source ? true,
   hackrf,
+  hermes_source ? true,
   limesdr_source ? true,
   limesuite,
-  perseus_source ? false, # needs libperseus-sdr, not yet available in nixpks
+  network_source ? true,
+  perseus_source ? false,
+  # needs libperseus-sdr, not yet available in nixpks
   plutosdr_source ? stdenv.hostPlatform.isLinux,
   libiio,
   libad9361,
   rfspace_source ? true,
   rtl_sdr_source ? true,
   rtl-sdr-osmocom,
-  libusb1, # osmocom better w/ rtlsdr v4
+  libusb1,
+  # osmocom better w/ rtlsdr v4
   rtl_tcp_source ? true,
   sdrplay_source ? false,
   sdrplay,
+  sdrpp_server_source ? true,
   soapy_source ? true,
   soapysdr-with-plugins,
+  spectran_http_source ? true,
   spyserver_source ? true,
   usrp_source ? false,
   uhd,
   boost,
-
   # Sinks
   audio_sink ? true,
   rtaudio,
   network_sink ? true,
   portaudio_sink ? false,
   portaudio,
-
   # Decoders
+  atv_decoder ? true,
+
+  # https://github.com/AlexandreRouma/SDRPlusPlus/issues/1511
+  # , dab_decoder ? false
   falcon9_decoder ? false,
+  kg_sstv_decoder ? false,
   m17_decoder ? false,
   codec2,
   meteor_demodulator ? true,
+  pager_decoder ? true,
   radio ? true,
-  weather_sat_decoder ? false, # is missing some dsp/pll.h
+  weather_sat_decoder ? false,
 
+  # is missing some dsp/pll.h
   # Misc
   discord_presence ? true,
   frequency_manager ? true,
+  iq_exporter ? true,
   recorder ? true,
+  rigctl_client ? true,
   rigctl_server ? true,
   scanner ? true,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "sdrpp";
 
   # SDR++ uses a rolling release model.
   # Choose a git hash from head and use the date from that commit as
   # version qualifier
-  git_hash = "27ab5bf3c194169ddf60ca312723fce96149cc8e";
-  git_date = "2024-01-22";
-  version = "1.1.0-unstable-" + git_date;
+  version = "unstable-2025-10-06";
```

I believe this should be `1.2.1-unstable-2025-10-06`, according to [pkgs/README.md#Versioning](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#Versioning) "It must start with a digit.", and my proposed change is the standard.

Furthermore, the selected commit is from October 5th I believe, and there is a more recent one from October 9th (why not use this one)?

```suggestion
  version = "1.2.1-unstable-2025-10-05";
```

---

Thanks, I had this PR pending for a while and missed the very last commit. I'll change it.